akka-actor-testkit-typed/src/main/scala/akka/actor/testkit/typed/scaladsl/BehaviorTestKit.scala

```diff
@@ -53,6 +53,11 @@ trait BehaviorTestKit[T] {
    */
   def childInbox[U](name: String): TestInbox[U]
 
+  /**
```

Comment:
add same in javadsl?

Reply:
@raboof this PR looks like it's done, or were we uncertain about the approach for children in BehaviorTestKit?

javadsl remaining?

Reply:
good